src/MySqlConnector/Protocol/Serialization/ByteBufferWriter.cs

```diff
@@ -126,6 +129,30 @@ public void Write(ReadOnlySpan<char> chars, bool flush)
 		}
 	}
 
+	public void WriteAscii(ReadOnlySpan<char> chars, bool flush)
```

Comment:
Is this ever called with `flush: false`? If not, the parameter could be removed and the code simplified.